test/e2e/16_fluxctl_sync.bats

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bats
+
+clone_dir=""
+
+function setup() {
+  load lib/env
+  load lib/install
+  load lib/poll
+  load lib/defer
+
+  kubectl create namespace "$FLUX_NAMESPACE"
+  # Install flux and the git server, allowing external access
+  install_git_srv git_srv_result
+  # shellcheck disable=SC2154
+  export GIT_SSH_COMMAND="${git_srv_result[0]}"
+  # Teardown the created port-forward to gitsrv and restore Git settings.
+  defer kill "${git_srv_result[1]}"
+
+  install_flux_with_fluxctl '15_fluxctl_sync'
+
+  # Clone the repo
+  clone_dir="$(mktemp -d)"
+  defer rm -rf "'$clone_dir'"
+  git clone -b master ssh://git@localhost/git-server/repos/cluster.git "$clone_dir"
+  # shellcheck disable=SC2164
+  cd "$clone_dir"
+}
+
+@test "fluxctl sync" {
+
+  # Sync
+  poll_until_true 'fluxctl sync succeeds' "fluxctl --k8s-fwd-ns ${FLUX_NAMESPACE} sync"
```

Comment:
IIRC this command returns the short commit hash of the applied commit, wouldn't that be better to use in the comparison below?

Reply:
Uhm I just want to make sure that sync tag is aligned with the latest commit. I think the test below is enough (and it doesn't require extra parsing).